html/mix-manifest.json

```diff
@@ -1,13 +1,26 @@
 {
-    "/js/app.js": "/js/app.js?id=50f7cb634569cc8bce2c",
+    "/js/app.js": "/js/app.js?id=7326d976853ee518f0dc",
     "/js/manifest.js": "/js/manifest.js?id=411da0f32dfa6d682e04",
     "/css/app.css": "/css/app.css?id=996b9e3da0c3ab98067e",
-    "/js/vendor.js": "/js/vendor.js?id=54e44dd06cb8f6a3e6fe",
+    "/js/vendor.js": "/js/vendor.js?id=00bcb49d05c1545efab0",
+    "/js/lang/ar.js": "/js/lang/ar.js?id=867496e0e07ff6097886",
     "/js/lang/de.js": "/js/lang/de.js?id=db973f6aaff0cda764c6",
-    "/js/lang/en.js": "/js/lang/en.js?id=1b381cf5180cf72bee65",
+    "/js/lang/en.js": "/js/lang/en.js?id=91a67cbff6e1f0da5cbc",
+    "/js/lang/es.js": "/js/lang/es.js?id=56158b47981e04b2eb54",
+    "/js/lang/et.js": "/js/lang/et.js?id=052eaaf24ae9a72474ff",
+    "/js/lang/fa.js": "/js/lang/fa.js?id=dc8b8eaf5c45ca5f1872",
     "/js/lang/fr.js": "/js/lang/fr.js?id=acd514ce666ebf45ef7b",
+    "/js/lang/gr.js": "/js/lang/gr.js?id=5c68b99a4e926052dce1",
+    "/js/lang/id.js": "/js/lang/id.js?id=6c3ef97eeb2cd2818885",
     "/js/lang/it.js": "/js/lang/it.js?id=b28a63928155eeb4e2a1",
+    "/js/lang/nl.js": "/js/lang/nl.js?id=052c04cc5d1f408fcf55",
+    "/js/lang/pl.js": "/js/lang/pl.js?id=0cca1dde319e96cac73d",
+    "/js/lang/pt-br.js": "/js/lang/pt-br.js?id=cf9e6e0244e1a17fddf1",
+    "/js/lang/pt.js": "/js/lang/pt.js?id=38b658d49f91c95b2566",
+    "/js/lang/ro.js": "/js/lang/ro.js?id=55032462a25b893fa186",
     "/js/lang/ru.js": "/js/lang/ru.js?id=f6b7c078755312a0907c",
+    "/js/lang/th.js": "/js/lang/th.js?id=3d5d6b9874b3ea673b3f",
+    "/js/lang/tr.js": "/js/lang/tr.js?id=d964e6a3e0bc51bbd7a1",
```

Comment:
where did all these come from?

Reply:
It's odd, when you run npm it creates hashes for more files than lnms translation:generate.  It is harmless however.  The hash is only used for cache breaking.